src/style.css

```diff
@@ -0,0 +1,77 @@
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  font-family: Arial, sans-serif;
+  color: rgb(255, 255, 255);
+  font-size: 100px;
+
+  width: 100%;
+  height: 300px;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+@media (min-width: 600px) and  (max-width: 899px) {
+  .block {
+    width: 50%;
+  }
+
+  .block-3 {
+    order: 1;
+  }
+
+  .block-5,
+  .block-6 {
+    order: 2;
+  }
+}
+
+@media (min-width: 900px) {
+  .block {
+    width: 33.33%;
+  }
```

Comment:
33.33 * 3 is 99.99, not 100

![Screenshot 2022-03-22 at 16 11 55](https://user-images.githubusercontent.com/79282749/159501675-52c7a176-9ec1-4d95-bc15-2ebb0c9ed544.jpg)
![Screenshot 2022-03-22 at 16 12 02](https://user-images.githubusercontent.com/79282749/159501712-d15b5c0f-c798-447f-a8e5-44fdbf152661.jpg)

it's a minor thing, but you should be aware of that. any idea how to make it 33.3333.....%?
also, because we are learning flex, let's use `flex-basis`